packages/react/src/__tests__/ReactChildren-test.js

```diff
@@ -888,7 +988,7 @@ describe('ReactChildren', () => {
     ]);
   });
 
-  it('does not warn for flattened positional children', async () => {
+  it('does not warn for flattened static children without keys', async () => {
```

Comment:
I realized while reading more code that we refer to these as "static" children, so this is just a rename.